developer-docs/how-to-guide/How_to_upload_existing_content_using_sunbird.md

```diff
@@ -9,286 +9,241 @@ allowSearch: true
-    <td>String</td>
-    <td>To make use of the API, you require authorization. Raise a request to the administrator for the use of the API. You will receive the authorization key. Specify the key received. </td>
-    <td>api_key</td>
-    <td>Yes</td>
-  </tr>
-  <tr>
-    <td>ts</td>
+    <td>file</td>
     <td>String</td>
-    <td>Time Stamp at which upload request was sent.</td>
-    <td>2018-09-05T10:55:22Z</td>
+    <td> The name of the content file that you want to upload</td>
+    <td>sample.pdf</td>
     <td>Yes</td>
   </tr>
 </table>
 
 
 #### Response Body (Upload Content API)
 
+    ```
     {
-   
        "id": "xyz.learning.content.upload",
-       
        "ver": "3.0",
-       
        "ts": "2018-09-05T10:55:22ZZ",
-       
        "params": {
-       
            "resmsgid": "d6dui5d1-60u8-4a79-23f9-78796195c2h7",
-           
            "msgid": null,
-           
            "err": null,
-           
            "status": "successful",
-           
            "errmsg": null
        },
-       
        "responseCode": "OK",
-       
        "result": {
            "content_url": "https://xyz-public-prod.s3-ap-south-1.amazonaws.com/content/do_112534095400542340156583/artifact/highq_153614492126723.pdf",
-
            "node_id": "do_112534095400542340156583",
-           
            "versionKey": "153614492126723"
-
        }
-
     }
+    ```
 
-For the youtube content or file url, pass it as file Url in <b>Upload API</b> or set the artifact Url of the content using  [Update Content API**](http://www.sunbird.org/apis/content/#operation/Update%20Content)
-
-After uploading the content, publish the content using **[Publish Content API**](http://www.sunbird.org/apis/content/#operation/Publish%20Content). 
-
-#### Request Body (Publish Content API)
-
-    {
-
-     "request": {
+For the YouTube content or file url, pass it as a `fileUrl` in [Upload API]() or set the artifactUrl of the content using [Update Content API](http://www.sunbird.org/apis/content/#operation/Update%20Content)
 
-     "content": {
 
-       "lastPublishedBy": "sunbird",
+### Publishing Content using API #### 
 
-       "publishChecklist": [
-           "No Hate speech, Abuse, Violence,Profanity"
-           "No Vulgarity",
-           "No Discrimination or Defamation",
-           "Is suitable for children",
-           "Appropriate Title, Description",
-           "Correct Board, Grade, Subject, Medium",
-           "Appropriate tags such as Resource Type, Concepts",
-           "Relevant Keywords",
-           "Content plays correctly",
-           "Can see the content clearly on Desktop and App",
-           "Audio (if any) is clear and easy to understand",
-           "No Spelling mistakes in the text",
-           "Language is simple to understand"
-       ]
+After uploading the content, publish the content using [Publish Content API](http://www.sunbird.org/apis/content/#operation/Publish%20Content). 
 
-      }
-
-     }
+#### Request Body
```

Comment:
Please verify the request body. Seems to be incomplete/inaccurate.

Reply:
The sample values should be created as-per the scenario laid out at the beginning of the document. You can replace the data type with sample values.

Reply:
Checked. The Publish checklist is configurable. In the scenario outlined, the values provided seem appropriate. @kochhar in case the values do not meet your expectations, request you to provide sample values.

Reply:
Please explain this in the doc to connect the checklist to the scenario

Reply:
Added note